doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -1079,13 +1079,12 @@ single full QuantumGraph file plus node numbers for each job. The default is
 using per-job QuantumGraph files.
 
 To use full QuantumGraph file, the submit YAML must set ``whenSaveJobQgraph`` to
-"NEVER" and the ``pipetask run`` command must include ``--qgraph-id {qgraphId}
---qgraph-node-id {qgraphNodeId}``.  For example:
+"NEVER" and the ``pipetask run`` command must include ``--qgraph-node-id {qgraphNodeId}``.  For example:
```

Comment:
Note to self:  probably time to deprecate the per-job QuantumGraph files code and documentation.  As far as I can tell even with our huge graphs today we're not having discussions about making per-job graph files.